sass-build/index.js

```diff
@@ -32,8 +34,10 @@ module.exports = function(options) {
   options = assign(defaultOptions, options);
 
   return gulp.src(options.src)
+    .pipe(debug ? sourcemaps.init() : noop())
```

Comment:
This line throws because `debug` is not defined. I think this should read
`.pipe(options.debug ? ...`